---
title: MSSQLSERVER_26013 | Microsoft-Dokumentation
ms.custom: ''
ms.date: 03/06/2017
ms.prod: sql-server-2014
ms.reviewer: ''
ms.suite: ''
ms.technology:
- database-engine
ms.tgt_pltfrm: ''
ms.topic: article
helpviewer_keywords:
- 26013 (Database Engine error)
ms.assetid: 7362fca4-b005-4872-bdfa-a3e32f834db8
caps.latest.revision: 9
author: craigg-msft
ms.author: craigg
manager: jhubbard
ms.openlocfilehash: 79d3c5a9b5e9499d2b9ddd5aea92c62844615ef9
ms.sourcegitcommit: 5dd5cad0c1bbd308471d6c885f516948ad67dfcf
ms.translationtype: MT
ms.contentlocale: de-DE
ms.lasthandoff: 06/19/2018
ms.locfileid: "36060954"
---
# <a name="mssqlserver26013"></a>MSSQLSERVER_26013
    
## <a name="details"></a>Details  
  
|||  
|-|-|  
|Produktname|SQL Server|  
|Ereignis-ID|26013|  
|Ereignisquelle|MSSQLSERVER|  
|Komponente|SQLEngine|  
|Symbolischer Name|SNI_SSL_READY|  
|Meldungstext|Das Zertifikat [Cert Hash(sha1) "%hs"] wurde für die Verschlüsselung erfolgreich geladen.|  
  
## <a name="explanation"></a>Erklärung  
 Diese Meldung dient zu Informationszwecken. Das genannte Zertifikat wurde erfolgreich in [!INCLUDE[ssNoVersion](../../includes/ssnoversion-md.md)] geladen und kann für die Verschlüsselung verwendet.  
  
## <a name="user-action"></a>Benutzeraktion  
 Es ist keine Aktion erforderlich.  
  
  